tests/php/PHPStanBearTest.py

```diff
@@ -0,0 +1,23 @@
+from bears.php.PHPStanBear import PHPStanBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+good_file = """
+<?php
+    $a = 1;
+?>
+"""
+
+
+bad_file = """
+<?php
+    $a =
+?>
+"""
+
+
+PHPStanBearTest = verify_local_bear(
```

Comment:
@satwikkansal @Mixih Most of the bear tests which i have seen are using `check_validity` and `verify_local_bear`.Till now i haven't seen a bear test using `check_results` so I'm really not that much comfortable with it.Decide guys :stuck_out_tongue: